### ISSUE TITLE
Horizontal Scroll On Desktop Only On Bottom of Table

### ISSUE THREAD

**Issue by Evitcani** [ujh/fountainpencompanion#2104]
In the desktop view, the horizontal scroll bar of the table is only at the bottom of the table. If there are many cluster tags, this obscures the editing button, making it a tedious task. 

Ideally, the scroll bar should always be present at the bottom of the visible area.

**Comment by ujh** [comment]
Yeah, the table isn't all that great once it overflows. 😞 I will see what I can do about it.
